scripts/hf_eval.py

```diff
@@ -1,10 +1,27 @@
 import torch
 
 from transformers import AutoModelForCausalLM, AutoTokenizer
+try:
+    from lm_eval.models.huggingface import HFLM
+    from lm_eval.evaluator import evaluate
+    from lm_eval.tasks import get_task_dict
+except ImportError as e:
+    print("""
+Error: The 'lm_eval' module was not found.
+To install, follow these steps:
 
-from lm_eval.models.huggingface import HFLM
-from lm_eval.evaluator import evaluate
-from lm_eval.tasks import get_task_dict
+1. Clone the repository:
```

Comment:
Does this work? `pip install git+https://github.com/EleutherAI/lm-evaluation-harness.git`

Reply:
yup